### ISSUE TITLE
Scroll wheel to zoom when in zoom/pan mode.

### ISSUE THREAD

**Issue by ZacWalk** [diffractor/diffractor#20]
When the user has zoom/pan mode; the mouse wheel should allow zooming the shown photo in or out. 
If the user holds control-shift and mouse wheel this could trigger entering zoom/pan mode.
When not in this mode the mouse wheel is used to scroll or change the thumbnail size when control is pressed.

**Comment by Sethycakes** [comment]
Yes please! As it stands, the choice seems to only be between scaled and unscaled viewing. Being able to customize the zoom level is a much-needed feature!
